tests/market/test_closed_markets.py

```diff
@@ -0,0 +1,127 @@
+import pytest
+import re
+import vega_sim.api.governance as governance
+from playwright.sync_api import Page, expect
+from fixtures.market import setup_continuous_market
+from conftest import init_vega
+
+
+@pytest.fixture(scope="class")
+def vega():
```

Comment:
why do we need this ?

Reply:
This is so we can reuse the same instance of vega across each class of tests